cluster/addons/dns/kube2sky/README.md

```diff
@@ -17,11 +17,13 @@ description of `-domain` below.
 `-domain`: Set the domain under which all DNS names will be hosted.  For
 example, if this is set to `kubernetes.io`, then a service named "nifty" in the
 "default" namespace would be exposed through DNS as
-"nifty.default.kubernetes.io".
+"nifty.default.svc.kubernetes.io".
+
+`--enable-pod-dns`: Allows pods to get dns entries based upon the `-domain` flag. If domain is set to `kubernetes.io`, then a pod named "niftypod" in the "default" namespace would be exposed through DNS as "niftypod.default.pod.kubernetes.io" 
```

Comment:
The code in kube2sky.go exposes "santized-ip.default.pod.kubernetes.io", can you re-phrase here too?